modules/cpmso-yaml-provider-spec-aws.adoc

```diff
@@ -0,0 +1,81 @@
+// Module included in the following assemblies:
+//
+// * machine_management/control-plane-machine-management.adoc
+
+:_content-type: REFERENCE
+[id="cpmso-yaml-provider-spec-aws_{context}"]
+= Sample AWS provider specification
+
+The provider specification must match that of the control plane machines created by the installation program. You can omit any field that is set in the failure domain section of the CR.
+// "that of the control plane machines" -> i.e., in their `machine` resource? We should explain how to reference this regardless.
+// "any field that is set in the failure domain section" = ? I don't see any overlap in the sample I have. Both have a placement.region, but this one is different than the ones for eh availability zones in the failure domain snippet.
+//In general, I'm not super sure what needs to be explained here, what a user can change, etc.
```

Comment:
So in the examples we've already omitted `subnet` and `placement.availabilityZone`, but when they look at a real Machine providerSpec, those will be configured. We take those values from the failure domains and inject them into the CPMS providerSpec to create the providerSpec for the Machines we create

Reply:
aha - should have guessed I didn't see any _because_ they were already omitted